app/posts/publish-teacher-training-courses/2020-10-01-uncoupling-UK-and-EU-course-fees.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Uncoupling UK and EU course fees
+description: Allowing providers to specify different course fees for UK and EU/international students
+date: 2020-10-01
```

Comment:
Needs updating?

Reply:
I date my entries to refelect an estimate of when the work was done. 👍